models/src/main/protobuf/CasperMessage.proto

```diff
@@ -20,10 +20,10 @@ service DeployService {
 }
 
 message DeployString {
-  bytes  user  = 1; //public key
-  int32  nonce = 2; //for uniqueness
-  string term  = 3; //rholang source code to deploy (will be parsed into `Par`)
-  bytes  sig   = 4; //signature of (hash(resource) + nonce) using private key
+  bytes  user      = 1; //public key
```

Comment:
Can we add a // TODO: Potentially remove if we are using ECDSA

Reply:
I personally don't mind there being some redundant information in messages. Redundancy gives you something to verify, a way to check the message has not been corrupted. I also think giving freedom in the signature algorithm, as we do with blocks is a good idea.